app/indexers/work_indexer.rb

```diff
@@ -169,6 +169,22 @@ class WorkIndexer < Kithe::Indexer
           end
         end
       end
+
+      if rec.members.any? {|mem| mem.asset? && mem.english_translation.present?}
+        acc << rec.members.map {|mem| mem.english_translation if mem.asset? }.compact.join(" ")
+      end
+      # Index the transcription into this if we can assume that the work is entirely in English.
+      if rec.members.any? {|mem| mem.asset? && mem.transcription.present?} && rec.language == ['en']
+        acc << rec.members.map {|mem| mem.transcription if mem.asset? }.compact.join(" ")
+      end
+    end
```

Comment:
Oh right, we do need to look through all members for this. 

Is this the first time we're needing to look at all members in indexing?  If so, that's definitely going to slow down indexing -- we need to make sure members are being Rails eager-loaded with `includes` to make it as efficient as possible, and then I'm still curious to know how much it's slowing it down, have some numbers on the PR.

Reply:
It does slow things down a bit, although not much. Will quantify.
I'd also rather use `pick`, since we're only looking at one or two values.

Reply:
I did mean `pluck`, yes. (`pick` does exist but that's not what I meant.)